### PR TITLE
updated workspaces and channels

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/GUI/Pages/Messenger/Workspace/Channels.razor [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+			@foreach (var channel in WorkspaceInfo.Channels)
+			{
+				<div class="cannels_item">
+					@channel.Name
+				</div>
+			}
+		}
+	</div>
+	<div class="channels_private">
+
```
Comment (Cheephot) [inline src/GUI/Pages/Messenger/Workspace/Channels.razor]: remove line

**Review by Cheephot — APPROVED** [review]

**Review by VLiamin — APPROVED** [review]

File: .editorconfig [hunk 2]
```diff
@@ -4,7 +4,7 @@
 ###############################
 # All files
 [*]
-indent_style = space
+indent_style =tab
```
Comment (BelehovEgor) [inline .editorconfig]: why did you change it?

Comment (BelehovEgor) [reply .editorconfig]: before we used space

File: src/GUI/Services/UserService.cs [hunk 3]
```diff
@@ -51,7 +51,8 @@ public async Task<string> GetUserNameAsync()
         null,
         null,
         null,
-        null);
+        null,
+				null);
```
Comment (BelehovEgor) [inline src/GUI/Services/UserService.cs:55]: fix

File: src/GUI/Services/UserService.cs [hunk 4]
```diff
@@ -87,7 +88,8 @@ public async Task<OperationResultResponseUserResponse> GetAuthorizedUserAsync(bo
         null,
         includeProjects,
         null,
-        null);
+        null,
+				null);
```
Comment (BelehovEgor) [inline src/GUI/Services/UserService.cs:92]: fix

**Review by BelehovEgor — APPROVED** [review]

File: src/GUI/Services/UserService.cs (hunk 3 above)
Comment (MariyaSin) [reply src/GUI/Services/UserService.cs:55]: in VS it have other view

File: src/GUI/Services/UserService.cs (hunk 4 above)
Comment (MariyaSin) [reply src/GUI/Services/UserService.cs:92]: in VS it has other view
